html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,12 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if ($i % 3 === 0 && $i % 5 === 0) {
+        echo'3の倍数であり、5の倍数'.'<br>';
+    } elseif ($i % 3 === 0) {
+        echo'3の倍数'.'<br>';
+    } elseif ($i % 5 === 0) {
+        echo'5の倍数'.'<br>';
+    } else {
+    echo $i.'<br>';
```

Comment:
```suggestion
        echo $i . '<br>';
```
インデント(字下げの深さ)をそろえましょう。
今後大量のコードを書くことになるとスタイルに気を付けるのが大変なので、vscodeの自動整形機能を使いましょう。
PHP Intelephense を vscodeにインストールすると勝手にインデント等を整形してくれるのでおすすめです。

![image](https://user-images.githubusercontent.com/22877094/84464625-02d5a980-acb0-11ea-9a61-a8bccee91e3d.png)